eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/GovSpec.hs

```diff
@@ -54,6 +55,25 @@ spec =
     policySpec
     networkIdSpec
     predicateFailuresSpec
+    unknownCostModelsSpec
+
+unknownCostModelsSpec ::
+  forall era.
+  ConwayEraImp era =>
+  SpecWith (ImpTestState era)
+unknownCostModelsSpec =
+  describe "Unknown CostModels" $ do
+    it "Accepts them" $ do
+      UnknownCostModels ukcm <- arbitrary
+      (drepC, hotCommitteeC, _gpi) <- electBasicCommittee
+      gai <-
+        submitParameterChange SNothing $
+          emptyPParamsUpdate
+            & ppuCostModelsL .~ SJust ukcm
+      submitYesVote_ (DRepVoter drepC) gai
+      submitYesVote_ (CommitteeVoter hotCommitteeC) gai
+      passNEpochs 2
+      getLastEnactedParameterChange `shouldReturn` SJust (GovPurposeId gai)
```

Comment:
Any particular reason to not also check that the enacted parameters are the ones generated ?

Reply:
Yes, we do need to verify that the updates wen't through successfully.